default_dial_option_server_option_test.go

```diff
@@ -26,6 +26,9 @@ import (
 	"google.golang.org/grpc/internal"
 )
 
+const maxRecvSize = 998765
+const initialWindowSize = 100
```

Comment:
Please make these local inside their tests.

Reply:
This is used 4/2 times, respectively?

Reply:
Actually this is in grpc I see I'll move to tests.